Dockerfile

```
@@ -44,3 +44,6 @@ LABEL io.k8s.description="driver-toolkit is a container with the kernel packages
       name="driver-toolkit" \
       io.openshift.release.operator=true \
       version="0.1"
+
+# Last layer for mapping the driver-toolkit to the corresponding Node     
+RUN  echo "{ \"KERNEL_VERSION\": \"${KERNEL_VERSION:-$(rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}"  kernel-core)}\", \"RT_KERNEL_VERSION\": \"${RT_KERNEL_VERSION:-$(rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}"  kernel-rt-core)}\", \"RHEL_VERSION\": \"${RHEL_VERSION}\" }" > /etc/driver-toolkit-release.json
```

```suggestion
RUN  echo "{ \"KERNEL_VERSION\": \"${KERNEL_VERSION:-$(rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}"  kernel-core)}\", \
\"RT_KERNEL_VERSION\": \"${RT_KERNEL_VERSION:-$(rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}"  kernel-rt-core)}\", \
\"RHEL_VERSION\": \"${RHEL_VERSION}\" }" > /etc/driver-toolkit-release.json
```
This looks good, but can we reformat it for readability?